WalletWasabi.Gui/Tabs/AboutViewModel.cs

```diff
@@ -16,5 +18,58 @@ public AboutViewModel(Global global) : base(global, "About")
 		public Version Version { get; }
 
 		public string VersionText => $"v{Version.ToString()}";
+
+		public void OnClearnetClicked()
+		{
+			OpenLink("https://wasabiwallet.io/");
+		}
+
+		public void OnTorClicked()
+		{
+			OpenLink("http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion");
```

Comment:
This is an onion address, browsers will not be able to solve the IP.

Reply:
If someone has a tor browser as their default browser, wouldn't it work then?